src/bounded.rs

```diff
@@ -132,13 +201,9 @@ impl<T> Bounded<T> {
                 }
             } else if stamp.wrapping_add(self.one_lap) == tail + 1 {
```

Comment:
why does this work with `tail + 1` and not use something like what's done for `new_tail`?

Reply:
This is done in order to tell if the stamp has advanced to where the tail would *logically* be, as opposed to where the tail actually is.

I'm not 100% clear on the logic here; this was written long before I joined smol-rs, and I think it was originally part of crossbeam.

Reply:
ok, it mght be a good idea to maybe investigate and properly document this sometime (not even necessarily inside of the code, a separate tex document with diagrams of this might be a good idea). Maybe I find time to do that (but unlikely in the next few months because I need to time to write a Bachelor thesis).